src/components/Beers.js

```diff
@@ -0,0 +1,34 @@
+import axios from "axios";
+import { useState } from "react";
+import { useEffect } from "react";
+
```

Comment:
If you are importing from the same place, you can combine titles like this:

import { useState, useEffect } from "react";